components/atom/tag/src/styles/index.scss

```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
         fill: $bgc;
       }
 
+      & > #{$self}-closeable {
```

Comment:
```suggestion
      & > #{$self}-action {
```

Reply:
This one refers to the class sui-AtomTag-closeable I can't change the name

Reply:
https://github.com/SUI-Components/sui-components/blob/b22b29db13ed763aeb08f9d39ad24acf3bf0309c/components/atom/tag/src/Standard.js#L30